examples/paper_experiments/README.md

```diff
@@ -0,0 +1,71 @@
+This folder contains the data processing and code to reproduce the graphstorm paper results.
```

Comment:
As the paper is not published. We should say:
This folder contains the data processing and code to profile GraphStorm framework on Amazon Review dataset.